clients/drcachesim/tests/burst_flush_aarch64.cpp

```diff
@@ -37,39 +38,73 @@
 #include <assert.h>
 #include <iostream>
 #include <signal.h>
+#include <setjmp.h>
+
+static sigjmp_buf mark;
+static int handled_sigill_count = 0;
 
 bool
 my_setenv(const char *var, const char *value)
 {
-#ifdef UNIX
     return setenv(var, value, 1 /*override*/) == 0;
-#else
-    return SetEnvironmentVariable(var, value) == TRUE;
-#endif
 }
 
 void
 sigill_handler(int signal)
 {
-    dr_app_stop_and_cleanup();
-    std::cerr << "all done\n";
-    _Exit(0);
+    handled_sigill_count++;
+    siglongjmp(mark, handled_sigill_count);
+    exit(-1);
 }
 
 static void
-do_some_work()
+dc_ivac()
```

Comment:
Either the longjmp should be inside this function, or there should be a comment here saying that this function raises SIGILL and it's the caller's responsibility to handle it.

Reply:
Added the comment.